account_purchase_stock_report_non_billed/wizard/account_sale_stock_report_non_billed_wiz.py

```diff
@@ -14,7 +14,6 @@ def _get_search_domain(self):
             [
                 domain,
                 [
-                    ("date_done", "<=", self.date_check),
```

Comment:
Why removing this? Both date intervals should be active.

Reply:
Because dates are being filtered at this level: https://github.com/OCA/account-financial-reporting/pull/945/files#diff-6c1eabc073e5d845d186043523e0ffcd942e59098613bee69886297ef8fbd937R77-R83

Reply:
And why 2 steps? Can't it be done everything in one shot?